ElectronClient/app/style.css

```diff
@@ -149,3 +149,52 @@ a {
 	outline: 0 none;
 }
 
+.modal-message {
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	color: grey;
+	font-size: 1.2em;
+	margin: 40px 20px;
+}
+
+.modal-message #loading-animation {
+	margin-right: 20px;
+  width: 20px;
+  height: 20px;
+  border: 5px solid lightgrey;
+  border-top: 4px solid white;
+  border-radius: 50%;
+  -webkit-transition-property: -webkit-transform;
+  -webkit-transition-duration: 1.2s;
+  -webkit-animation-name: rotate;
+  -webkit-animation-iteration-count: infinite;
+  -webkit-animation-timing-function: linear;
+
+  -moz-transition-property: -moz-transform;
+  -moz-animation-name: rotate;
+  -moz-animation-duration: 1.2s;
+  -moz-animation-iteration-count: infinite;
+  -moz-animation-timing-function: linear;
+
+  transition-property: transform;
+  animation-name: rotate;
+  animation-duration: 1.2s;
+  animation-iteration-count: infinite;
+  animation-timing-function: linear;
+}
```

Comment:
Could you try to remove all the prefixed CSS (-webkit-*, -moz-*). I believe it should still works since Electron supports the regular properties and that would make the code shorter.